src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityNetPeers.cs

```diff
@@ -0,0 +1,38 @@
+//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+// 
+
+using System.Linq;
+using Nethermind.Network;
+
+namespace Nethermind.JsonRpc.Modules.Parity
+{
+    public class ParityNetPeers
+    {
+        public int Active { get; set; }
+        public int Connected { get; set; }
+        public int Max { get; set; }
+        public PeerInfo[] Peers { get; set; }
+        
+        public ParityNetPeers(IPeerManager peerManager)
+        {
+            Active = peerManager.ActivePeers.Count;
+            Connected = peerManager.ConnectedPeers.Count;
+            Max = peerManager.MaxActivePeers;
+            Peers = peerManager.ActivePeers.Select(p => new PeerInfo(p)).ToArray();
+        }
+    }
```

Comment:
ParityNetPeers shouldn't rely on IPeerManager, maybe abstract it to some builder method

Reply:
Why not @LukaszRozmej ?

Reply:
I see

Reply:
Yeah, this should not be in the DTO